docs/CONTRIBUTOR_GUIDE.md

```diff
@@ -0,0 +1,55 @@
+# Contributor Guide
+
+This guide is intended for people who want to start working on `krew` itself.
+If you intend to write a new plugin, see [Developer Guide](./DEVELOPER_GUIDE.md) instead.
+
+## Setting up the environment
+
+Krew is built with go 1.10, but newer versions will do as well.
+Most toolchains will expect that the krew repository is on the `GOPATH`.
+To set it up correctly, do
+```bash
```

Comment:
would be good to add an empty line between paragraphs vs titles vs code-fences vs lists in general.